tldw_Server_API/app/core/MCP_unified/tests/test_slides_module.py

```
@@ -113,6 +114,44 @@ def generate_from_text(self, source_text, title_hint=None, provider=None, model=
         return {"title": title_hint or "Generated", "slides": [{"order": 0, "title": "Slide", "content": "Content"}]}
 
 
+def test_slides_module_get_media_content_uses_managed_media_database(monkeypatch, tmp_path):
+    mod = SlidesModule(ModuleConfig(name="slides"))
+    events = []
+
+    class _FakeDb:
+        def get_media_by_id(self, media_id):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `get_media_by_id`

(ANN202)